bin/start

```diff
@@ -35,6 +35,12 @@ export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
 export WEB_MEMORY=$WEB_MEMORY
 export WEB_CONCURRENCY=$WEB_CONCURRENCY
 
-export DYNO="${PROCESS:-run}.$(hostname)"
+if [ -f "${ECS_CONTAINER_METADATA_FILE:-}" ]
```

Comment:
is the `:-` extraneous here?

Reply:
Nope, required

Reply:
because of bash `set -o nounset` above! got it